docs/source/notebooks/03_Extract_geographical_watershed_properties.ipynb

```diff
@@ -9,7 +9,9 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    str(latitude)
</pre><p>using <code>print</code>  (which does an implicit <code>str</code>  conversion) would probably work the same here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/raven/pull/422/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>